### PR TITLE
read region, cluster name from variables

### PR DESCRIPTION
**Pull request by rasheedamir**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -71,7 +72,8 @@ destroy_all: \
 	destroy_iam \
 	destroy_route53 \
 	destroy_s3 \
-	destroy_vpc
+	destroy_vpc \
+	destroy_helloworld
```
Comment (rasheedamir) [inline Makefile]: @muhammadIkramUlHaq seems like you took branch from helloworld & not master!

**Comment by rasheedamir** [comment]
@muhammadIkramUlHaq @hazim1093 please remove **helloworld** module from this PR and changeset

File: resources/cloud-config/files.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ write_files:
         AWS_CONFIG_ENV=/root/.aws/envvars
         source $AWS_CONFIG_ENV
         IMAGE=suet/awscli:latest
-        APPBUCKET=s3://${AWS_ACCOUNT}-coreos-cluster-config/apps
+        APPBUCKET=s3://${AWS_ACCOUNT}-CLUSTER-NAME-config/apps
```
Comment (rasheedamir) [inline resources/cloud-config/files.yaml:25]: @hazim1093 have you verified in that this variable name is replaced?

File: resources/cloud-config/files.yaml [hunk 3]
```diff
@@ -76,5 +76,5 @@ write_files:
   - path: /etc/systemd/system/docker.service.d/50-insecure-registry.conf
     content: |
         [Service]
-        Environment=DOCKER_OPTS='--insecure-registry=10.0.0.0/8,dockerhub.coreos-cluster.local'
+        Environment=DOCKER_OPTS='--insecure-registry=10.0.0.0/8,dockerhub.CLUSTER-NAME.local'
```
Comment (rasheedamir) [inline resources/cloud-config/files.yaml:79]: @hazim1093 have you verified in that this variable name is replaced?

File: resources/cloud-config/files.yaml (hunk 2 above)
Comment (hazim1093) [inline resources/cloud-config/files.yaml]: yes sir verified

Comment (hazim1093) [inline resources/cloud-config/files.yaml:25]: @rasheedamir yes, verified

File: resources/cloud-config/files.yaml (hunk 3 above)
Comment (hazim1093) [inline resources/cloud-config/files.yaml:79]: @rasheedamir verified too!

**Pull request merged**